cmd/cluster/cpd.go

```diff
@@ -89,32 +94,38 @@ func (o *cpdOptions) run() error {
 		return fmt.Errorf("this command doesn't support GCP yet. Needs manual investigation:\nocm backplane cloud console -b %s", o.clusterID)
 	}
 
-	fmt.Println("Generating AWS credentials for cluster")
-	// Get AWS credentials for the cluster
-	awsClient, err := osdCloud.GenerateAWSClientForCluster(o.awsProfile, o.clusterID)
-	if err != nil {
-		fmt.Println("PLEASE CONFIRM YOUR CREDENTIALS ARE CORRECT. If you're absolutely sure they are, send this Service Log https://github.com/openshift/managed-notifications/blob/master/osd/aws/ROSA_AWS_invalid_permissions.json")
-		fmt.Println(err)
-		return err
-	}
+	if isolated, err := isIsolatedBackplaneAccess(cluster, ocmClient); err != nil {
+		return fmt.Errorf("unable to determine which backplane flow this cluster is using: %w.\nNeeds manual investigation:\nocm backplane cloud console -b %s", err, o.clusterID)
+	} else if isolated {
+		return fmt.Errorf("this command doesn't support the isolated backplane flow yet. Needs manual investigation:\nocm backplane cloud console -b %s", o.clusterID)
```

Comment:
Would it be possible for us to somehow still invoke the old flow in this case? Or does this require the new flow?

Reply:
Invoking the old flow on a cluster using the isolated flow is guaranteed to fail, which could lead us to erroneously telling the SRE to send a service log to the customer.